pkg/templates/cel/internal/params/gen-params.go

```diff
@@ -0,0 +1,68 @@
+// Code generated by kube-linter template codegen. DO NOT EDIT.
+//go:build !templatecodegen
+// +build !templatecodegen
+
+package params
+
+import (
+	"fmt"
+	"strings"
+
+	"golang.stackrox.io/kube-linter/pkg/check"
+	"golang.stackrox.io/kube-linter/pkg/templates/util"
+)
+
+var (
+	// Use some imports in case they don't get used otherwise.
+	_ = util.MustParseParameterDesc
+
+	checkParamDesc = util.MustParseParameterDesc(`{
+	"Name": "check",
+	"Type": "string",
+	"Description": "Check is a CEL expression used to validate the subject and objects.",
+	"Examples": null,
+	"Enum": null,
+	"SubParameters": null,
+	"ArrayElemType": "",
+	"Required": false,
+	"NoRegex": false,
+	"NotNegatable": false,
+	"XXXStructFieldName": "Check",
+	"XXXIsPointer": false
+}
+`)
+
+	ParamDescs = []check.ParameterDesc{
+		checkParamDesc,
+	}
+)
+
+func (p *Params) Validate() error {
+	var validationErrors []string
+	if len(validationErrors) > 0 {
+		return fmt.Errorf("invalid parameters: %s", strings.Join(validationErrors, ", "))
+    }
+	return nil
+}
```

Comment:
I don't quite understand what's the value of this validation, nor I see `check` 's CEL-specific validation here. Is it happening somewhere else or implicitly?